File: official/docs/node/reports/create_report.js
const Easypost = require('@easypost/api');

const api = new Easypost(process.env.EASYPOST_API_KEY);

const report = new api.Report({
  start_date: '2022-10-01',
  end_date: '2022-10-31',
});

report.save().then(console.log);

File: official/docs/go/reports/create_report.go
package example

import (
	"fmt"
	"os"

	"github.com/EasyPost/easypost-go/v2"
)

func main() {
	apiKey := os.Getenv("EASYPOST_API_KEY")
	client := easypost.New(apiKey)

	reportOptions := &easypost.Report{
		StartDate: "2016-01-01",
		EndDate:   "2016-01-31",
	}

	report, _ := client.CreateReport(
		"shipment",
		reportOptions,
	)

	fmt.Println(report)
}
